packages/components/src/atoms/Price/Price.tsx

```diff
@@ -53,8 +53,8 @@ const Price = forwardRef<Omit<HTMLSpanElement, 'children'>, PriceProps>(
     },
     ref
   ) {
-    const formattedPrice = formatter(value, variant)
-
+    const formatedValue = Math.floor(value * 100)/100
```

Comment:
This should be a dumb component. It should not presume anything from the value. Move this logic to the store.